samza-test/src/main/java/org/apache/samza/test/framework/TestRunner.java

```diff
@@ -267,22 +271,34 @@ public TestRunner addOutputStream(CollectionStream stream) {
     return this;
   }
 
+
   /**
    * Utility to run a test configured using TestRunner
+   *
+   * @param timeout time to wait for the high level application or low level task to finish. This timeout does not
+   *                include input stream initialization time or the assertion time over output streams.
+   *                This timeout just accounts for time that samza job takes run
```

Comment:
When timeout elapses, we throw up. It's better to mention that in the java doc.

Reply:
Added that